### PR TITLE
RK-698 - Aws regression test

### PR DESCRIPTION
**Pull request by ManuelNemirovsky**

### CODE REVIEW

File: python-aws-chalice/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM lambci/lambda:build-python2.7
+ 
+ADD . .
+
+# Install app dependencies
+RUN pip install --upgrade pip
+RUN pip install -r requirements.txt
+RUN pip install 'chalice == 1.7.0'
+
+RUN sed -i '5s/.*/    "ROOKOUT_TOKEN": "${ROOKOUT_TOKEN}",\n    "ROOKOUT_ROOK_TAGS": "python_chalice_lambda_regression_test",\n    "ROOKOUT_DEBUG": "1"/' .chalice/config.json
```
Comment (MickaelAlliel) [inline python-aws-chalice/Dockerfile]: remove, dockerfile does it

File: node-aws-lambda/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+regressionTest {
+    FILE_PATH =  "index.js"
+    LINE_NUM =  "4"
+    SHA =  "b2e71f93fecfa7b5795c1c0c1d21f115aa408c78da5a652a25d5132ee6e877f5"
+    URL_FOR_TRIGGER = "node_aws_lambda_regression_test"
+    LANG = "NodeJs"
+    BACKGROUND_IMAGE_CMD = "bash /../../awsTest.sh &"
```
Comment (MickaelAlliel) [inline node-aws-lambda/Jenkinsfile]: try /awsTest
/../.. is weird and hard to udnerstand

File: python-aws-chalice/runTest.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!bin/bash
+cd /var/task
+declare RESULT=($(chalice deploy)) ; \
+aws lambda update-function-configuration \
+            --function-name python_chalice_lambda_regression_test-dev \
+            --region us-east-2 \
+            --environment Variables="{ROOKOUT_TOKEN=$ROOKOUT_TOKEN,ROOKOUT_ROOK_TAGS=python_chalice_lambda_regression_test-dev,ROOKOUT_DEBUG=1,ROOKOUT_LOG_TO_STDERR=1}" 
+sleep 20
+curl ${RESULT[26]}
```
Comment (MickaelAlliel) [inline python-aws-chalice/runTest.sh]: comment with chalice issue url why custom dns is not possible

File: python-aws-lambda/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+ADD . . 
+
+# Install app dependencies
+RUN pip install --upgrade pip
+RUN pip install rook -t .
+
+FROM kramos/alpine-zip AS img2
+
+RUN apk add --update zip
```
Comment (MickaelAlliel) [inline python-aws-lambda/Dockerfile]: try to install/use zip in lambda:build-python img, ok if doesnt work

File: python-aws-lambda/Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,8 @@
+regressionTest {
+    FILE_PATH =  "lambda_function.py"
+    LINE_NUM =  "5"
+    SHA =  "33e7260e49e3ecc9a71b0596d193aaf49a685c3bde357c1e18785062ecfb45b6"
+    URL_FOR_TRIGGER = "python_lambda_regression_test"
+    LANG = "Python"
+    BACKGROUND_IMAGE_CMD = "bash /../../awsTest.sh &"
```
Comment (MickaelAlliel) [inline python-aws-lambda/Jenkinsfile]: same thing /../../..

File: python-aws-lambda/Dockerfile (hunk 4 above)
Comment (ManuelNemirovsky) [reply python-aws-lambda/Dockerfile]: DIdn't work
But I changed the path of some lines to root insted of `/../../`

**Pull request merged**